addons/website/static/src/js/content/inject_dom.js

```diff
@@ -15,6 +15,21 @@ export function unhideConditionalElements() {
     document.head.appendChild(styleEl);
     const conditionalEls = document.querySelectorAll('[data-visibility="conditional"]');
     for (const conditionalEl of conditionalEls) {
+        // For mega menu block, add conditional visibility to the navbar link
+        if (conditionalEl.parentElement.classList.contains("o_mega_menu")) {
+            const desktopMegaMenuToggleEl = conditionalEl.closest("li[role='presentation']");
```

Comment:
`li` element are a `dropdown` not a `toogle`

<img width="475" height="122" alt="Image" src="https://github.com/user-attachments/assets/008e1c03-09dc-400a-8236-438a49457771" />